python-avd/pyavd/_eos_designs/shared_utils/node_type_keys.py

```diff
@@ -174,6 +173,39 @@
                 "l3": True,
             },
         },
+        {
+            "key": "ptp_leaf",
```

Comment:
Should this not be prefixed with `media_` as well?

Reply:
would it be okay to leaf the media_ so non-media PTP customers can consume it too ?